kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -78,6 +79,9 @@
   public static final String ACL_CANNED_CONFIG = "s3.acl.canned";
   public static final String ACL_CANNED_DEFAULT = null;
 
+  public static final String COMPRESSION_TYPE_CONFIG = "s3.compression.type";
+  public static final String COMPRESSION_TYPE_DEFAULT = "none";
```

Comment:
Hi I'm not a maintainer but I feel like it should mimic avro compression and no compression should be named null. What do you thing about this?

Reply:
This is actually messier than I realized. The `compression.type` for producer client uses "none" for no compression, but the same option for broker and topic configs uses "uncompressed" for no compression. And the avro [CodecFactory](https://avro.apache.org/docs/1.8.1/api/java/org/apache/avro/file/CodecFactory.html) uses null.

The choice of "null" for avro is driven by conventions in the avro project. If someone is used to avro, it makes sense to keep the avro.codec option consistent with that terminology.

For this more general compression.type, I have a preference for following Kafka conventions over avro conventions.